script/refresh_sitemap

```diff
@@ -92,31 +92,39 @@ LOCALES = Language.connection.select_values(%(
   SELECT locale FROM languages WHERE NOT beta ORDER BY `order` ASC
 ))
 
+# Control behavior of crowlers that follow robots.txt rules
+# For documentation, see:
+# https://developers.google.com/search/docs/advanced/robots/robots_txt
+# http://www.robotstxt.org
+
+# Other ways for MO to block/control robots include:
+# - Manually block ip address ranges at the firewall
+# - Automatically block in Rails an anonymous user (including robots)
+#   that's hogging the webserver. See script/update_ip_stats.rb
+# - Automatically block in Rails a fake GoogleBot.
+#    See script/update_googlebots.rb
+# - Manually block in Rails an anonymous user via the Admin UI
+# - Limit them via browser gem and #bot? method
 ROBOTS_HEADER = <<~"TXT"
-  # For documentation, see:
-  # https://developers.google.com/search/docs/advanced/robots/robots_txt
-  # http://www.robotstxt.org
+
   Sitemap: #{SITEMAP_URL.sub("NAME", "index")}
 
-  # Completely disallowed bots
-  User-agent: ltx71
+  # Completely block all robots that are not explicitly whitelisted below
+  User-agent: *
+  Disallow: /
+
+  # Google AdBot crawlers must be named explicitly
+  # https://developers.google.com/search/docs/advanced/robots/create-robots-txt
   User-agent: AdsBot-Google
-  User-agent: AhrefsBot
-  User-agent: Applebot
-  User-agent: DAUM
-  User-agent: MauiBot
-  User-agent: MJ12bot
-  User-agent: SpiderLing
-  User-agent: spbot
+  User-agent: AdsBot-Google-Mobile
+  User-agent: AdsBot-Google-Mobile-Apps
   Disallow: /
 
-  # Rules applying to some allowed bots
+  # Whitelisted crawlers
+  User-agent: bingbot
   User-agent: CCBot
```

Comment:
CCBot is combined with other whitelisted bots so that ROBOTS_FOOTER directives will also apply to it.
Is that right?
A side-effect is that there is now a shorter delay for CCBot (15 sec vs 30 sec). I think we can live with that.